smarty/templates/login.tpl

```diff
@@ -191,7 +191,7 @@ $(document).ready(function() {
 		  <div class="panel-body">
 		  		<div class="col-xs-12">
 		  			<center>
-		  				<img src="images/LORIS_logo_141007.svg" class="img-responsive" alt="Responsive image" onerror="this.src='images/LORIS_Logo_141007.png'" align="middle" width="85%"> 
+		  				<img src="{$baseurl}/images/LORIS_logo_141007.svg" class="img-responsive" alt="Responsive image" onerror="this.src='{$baseurl}/images/LORIS_Logo_141007.png'" align="middle" width="85%"> 
 		  			</center>
 					<br>
 		  		</div>
```

Comment:
Those pages (links) are accessible from htdocs. I have not found any problem on a site based at localhost/ or somewhere else like localhost/something/something.